app/lib/bk/compat/parsers/circleci/orbs/docker.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require_relative 'docker/docker'
+
+module BK
+  module Compat
+    module CircleCISteps
+      DockerOrb = BK::Compat::Parsers::CircleCI::Orbs::Docker::Docker
```

Comment:
the way it was before the last commit was OK, why did you remove the class only to add it as a reference here?